setup.py

```diff
@@ -51,33 +55,23 @@ def get_version():
     set(itertools.chain.from_iterable(map(lambda group: extras[group], all_groups)))
 )
 
+version = get_version()
```

Comment:
This function doesn't seemed to be used and the setup version=get_version

Reply:
Fixed to do `version=version`.